src/store/reducers/capabilities/hooks.ts

```diff
@@ -68,3 +73,17 @@ export const useClusterDashboardAvailable = () => {
 export const useStreamingAvailable = () => {
     return useGetFeatureVersion('/viewer/query') >= 7;
 };
+
+const useGetSecuritySetting = (feature: SecuritySetting) => {
+    const database = useDatabaseFromQuery();
+
+    return useTypedSelector((state) => selectSecuritySetting(state, feature, database));
+};
+
+export const useAccessTotallyRestricted = () => {
+    return useGetSecuritySetting('UseLoginProvider') === false;
+};
```

Comment:
I don't like the naming, I'd better call it like `useLoginFormCannotBeUsed` or `useClusterWithoutAuthInUI`.

The same relates to `isForbidded` var name